src/constants.js

```diff
@@ -37,5 +37,7 @@ export default {
   UTM_MEDIUM: 'utm_medium',
   UTM_CAMPAIGN: 'utm_campaign',
   UTM_TERM: 'utm_term',
-  UTM_CONTENT: 'utm_content'
+  UTM_CONTENT: 'utm_content',
+
+  ATTRIBUTION_EVENT: 'attribution captured'
```

Comment:
open to feedback on a better event name

Reply:
maybe `DEBUG: attribution captured` or `$attributionCaptured` so it's clearer to our customers that this is autogenerated by the SDK and/or

Reply:
Updated to `$attributionCaptured`